pkg/github/github.go

```diff
@@ -121,9 +124,20 @@ func getAvailableReleases(
 	}
 
 	// Convert ReleaseInfo array to an array of just the version strings
-	releaseVersions := make([]string, len(releases))
-	for index, release := range releases {
-		releaseVersions[index] = release.TagName
+	releaseVersions := make([]string, 0)
+	for _, release := range releases {
+		versionStr := strings.TrimPrefix(release.Name, "v")
+		_, err := semver.NewVersion(versionStr)
+
+		if err != nil {
+			// Skip adding versions that don't follow semver
+			continue
+		} else if strings.Contains(releasesURL, "appliance") && (strings.Split(versionStr, ".")[0] == "11" || strings.Split(versionStr, ".")[0] == "10") {
```

Comment:
This seems really hacky - we don't use specific exclusions for any particular repo and inserting some very iffy logic about it here seems inappropriate.

Reply:
we don't tag the semver version AND versions that follow a separate similar looking versioning scheme in any project besides the appliance.

I'm open to suggestions, but the `v10.x` and `v11.x` streams in the appliance should be excluded from the calculations here. maybe we should just retag older ones as `PAS_10.x.0` and `PAS_v11.x.0` since that's the convention we use now, and then this exception could be dropped entirely?

Reply:
I added a comment to explain it. I'm also checking to see if I can clean up the tags in the appliance repo, but for now I'll leave it as-is with the comment that explains the purpose.

Reply:
eta: removed this entirely bc the old tags don't correspond to gh releases, so we ignore them anyway.